### ISSUE TITLE
Uncaught TypeError: Cannot read property 'stdin' of undefined

### ISSUE THREAD

**Issue by Mmmmh** [lsegal/atom-runner#101]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.2.3
**System**: Unknown Windows Version
**Thrown From**: [atom-runner](https://github.com/lsegal/atom-runner) package, v2.4.7

### Stack Trace

Uncaught TypeError: Cannot read property 'stdin' of undefined

```
At /C:/Users/Romain/.atom/packages/atom-runner/lib/atom-runner.coffee:275

TypeError: Cannot read property 'stdin' of undefined
    at AtomRunner.execute (file:///C:/Users/Romain/.atom/packages/atom-runner/lib/atom-runner.coffee:171:11)
    at AtomRunner.run (file:///C:/Users/Romain/.atom/packages/atom-runner/lib/atom-runner.coffee:108:6)
    at atom-workspace.<anonymous> (file:///C:/Users/Romain/.atom/packages/atom-runner/lib/atom-runner.coffee:69:57)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:\Users\Romain\AppData\Local\atom\app-1.2.3\resources\app.asar\src\command-registry.js:260:29)
    at C:\Users\Romain\AppData\Local\atom\app-1.2.3\resources\app.asar\src\command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (C:\Users\Romain\AppData\Local\atom\app-1.2.3\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:533:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (C:\Users\Romain\AppData\Local\atom\app-1.2.3\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:356:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeydown (C:\Users\Romain\AppData\Local\atom\app-1.2.3\resources\app.asar\src\window-event-handler.js:99:36)
    at HTMLDocument.<anonymous> (C:\Users\Romain\AppData\Local\atom\app-1.2.3\resources\app.asar\src\window-event-handler.js:3:61)
```

### Commands

```
 13x -3:31 core:backspace (atom-text-editor.editor.is-focused)
     -3:16.2.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -2:38.5.0 core:move-right (atom-text-editor.editor.is-focused)
  2x -2:35.1.0 core:backspace (atom-text-editor.editor.is-focused)
     -2:32.2.0 core:move-left (atom-text-editor.editor.is-focused)
     -2:31 core:move-right (atom-text-editor.editor.is-focused)
     -0:12.8.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:12.2.0 symbols-view:toggle-file-symbols (atom-text-editor.editor.is-focused)
     -0:03.2.0 run:file (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "one-light-ui",
      "one-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-runner, v2.4.7

# Dev
No dev packages
```

**Comment by willin** [comment]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.2.4
**System**: Mac OS X 10.11.2
**Thrown From**: [atom-runner](https://github.com/lsegal/atom-runner) package, v2.4.7


### Stack Trace

Uncaught TypeError: Cannot read property 'stdin' of undefined

```
At /Users/willin/.atom/packages/atom-runner/lib/atom-runner.coffee:171

TypeError: Cannot read property 'stdin' of undefined
    at AtomRunner.execute (/Users/willin/.atom/packages/atom-runner/lib/atom-runner.coffee:171:11)
    at AtomRunner.run (/Users/willin/.atom/packages/atom-runner/lib/atom-runner.coffee:108:6)
    at atom-workspace.<anonymous> (/Users/willin/.atom/packages/atom-runner/lib/atom-runner.coffee:69:57)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:533:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:356:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:99:36)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```

### Commands

```
     -1:30.8.0 settings-view:open (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-one-dark-syntax.theme-one-dark-ui)
     -1:29.7.0 core:select-all (atom-text-editor.editor.mini.is-focused)
     -1:28.4.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:35.6.0 tree-view:add-file (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
  2x -0:31.3.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:29.7.0 core:select-all (atom-text-editor.editor.mini.is-focused)
     -0:29.5.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:29.2.0 core:cancel (atom-text-editor.editor.mini.is-focused)
     -0:26.5.0 tree-view:add-file (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:23.7.0 core:confirm (atom-text-editor.editor.mini.is-focused)
  5x -0:20.3.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:14.2.0 tree-view:add-file (span.name.icon.icon-file-directory)
     -0:12.3.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:03.7.0 core:save (atom-text-editor.editor.is-focused)
     -0:02.7.0 atom-terminal-panel:toggle-autocompletion (atom-text-editor.editor.is-focused)
     -0:02.3.0 run:file (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
atom-beautify, v0.28.19
atom-runner, v2.4.7
atom-terminal-panel, v4.4.4
atom-ternjs, v0.10.2
autocomplete-paths, v1.0.2
color-picker, v2.0.13
docblockr, v0.7.3
file-icons, v1.6.12
javascript-snippets, v1.0.0
linter, v1.11.3
minimap, v4.16.2
terminal-panel, v1.14.1
tool-bar, v0.1.9
tool-bar-main, v0.0.8

# Dev
No dev packages
```

**Comment by lsegal** [comment]
Likely fixed by #101, without any steps to reproduce it's hard to know. Please re-open if you still have this issue **with reproduction steps**.
